lib/Partitioner/Partitioner.cpp

```diff
@@ -77,9 +77,11 @@ llvm::Error Partitioner::finalize(const DAGListTy &partitions,
 
   // Validate the functions after partitioning.
   for (Function *subF : module_->getFunctions()) {
-    RETURN_ERR_IF_NOT(subF->verify(),
-                      strFormat("Conversion led to invalid function: %s",
-                                subF->getName().str().data()));
+    if (!subF->verify()) {
```

Comment:
if you changed this so that you can add `GlowErr::ErrorCode::PARTITIONER_ERROR` I think you can do that with RETURN_ERR_IF_NOT (should work at least)